user-guide/Advanced_Modules/Automation_module/Special_parameters_available_in_DMS_Automation_scripts.md

```diff
@@ -15,7 +15,7 @@ The following table lists the special parameters available in Automation scripts
 | \<Redundancy Type> | 65003 | The type of redundancy group switch (only if the execution of the script was caused by a redundancy group switch).<br> Values:<br> -  SWITCH_OVER<br> -  SWITCH_BACK |
 | \<Redundancy Cause> | 65004 | The cause of a redundancy group switch (only if the execution of the script was caused by a redundancy group switch).<br> Values:<br> -  prioritychanged<br> -  manual<br> -  automatic |
 | \<Correlation State> | 65002 | Whether the Correlation rule was triggered because the trigger condition matched the alarm (“up”) or whether it was triggered because the trigger condition no longer matched the alarm (“down”) (only if the script was executed via a Correlation rule).<br> Values:<br> -  up<br> -  down |
-| \<Correlation Alarm ID> | 65005 | The ID of the alarm that triggered the Correlation rule (only if the script was executed via a Correlation rule). |
+| \<Correlation Alarm ID> | 65005 | [Obsolete] The ID of the alarm that triggered the Correlation rule (only if the script was executed via a Correlation rule). Avoid using this value. Extract the full alarm tree reference from \<Correlation Alarm Info> (65006) instead. |
```

Comment:
From which DataMiner versions onwards is this obsolete? The same documentation is used for all currently supported DataMiner versions, so we need to make sure that updates are done in such a way that they are not confusing for users of older versions.

Reply:
Any version really 🙂 The "alarm id" which is being passed along here is just a number and was already meaningless before without also using the matching dmaid info